src/com/gr1t/taskmanager/Task.java

```diff
@@ -1,15 +1,58 @@
 package com.gr1t.taskmanager;
 
 public class Task {
-    private int id;
+    private String title;
     private String description;
     private Priority priority;
-    private boolean isCompleted;
+    private boolean isCompleted = false;
 
-    public Task(boolean isCompleted, Priority priority, String description, int id) {
-        this.isCompleted = isCompleted;
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Priority getPriority() {
+        return priority;
+    }
+
+    public void setPriority(Priority priority) {
         this.priority = priority;
+    }
+
+    public boolean isCompleted() {
+        return isCompleted;
+    }
+
+    public void setCompleted(boolean completed) {
+        isCompleted = completed;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Getters and Setters and even toString: Cannot return a value from a constructor* when trying put constructor above them (as it should be), so it's at at the bottom)